### PR TITLE
[WEED-736] Parse Principal Point

### PR DESCRIPTION
**Pull request by njwitthoeft**
# Principal Point functionality
## What?
- Parse principal point from cameras with calibrated principal point information
## Why?
- To allow for more accurate single-image quicktiles from the 65R

## CR Questions
- Should `camera_params` now return this info as well? That would require a major version bump. Should that be done at the same time as optical distortions?
- I migrated to GHA, I can't get into travis and this was an easy fix. Should I split that out?

## PR Checklist
- [ ] Merged latest master
- [x] Updated version number
- [x] Version numbers match between package ``_version.py`` and *pyproject.toml*
## Breaking Changes
None.

### CODE REVIEW

File: imgparse/_version.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
 """Defines package version.  Parsed by setup.py and imported by __init__.py."""
 
-__version__ = "1.15.3"
+__version__ = "1.15.4"
```
Comment (elliott-imhoff) [inline imgparse/_version.py:3]: Minor version bump for the new functionality

File: imgparse/imgparse.py [hunk 2]
```diff
@@ -218,6 +218,35 @@ def get_focal_length(image_path, exif_data=None, xmp_data=None, use_calibrated=F
...
+        pt = list(map(float, str(xmp_data[xmp_tags.PRINCIPAL_POINT]).split(",")))
+        return PixelCoords(x=pt[0], y=pt[1])
+    except KeyError:
+        raise ParsingError(
+            "Couldn't find the principal point tag. Sensor might not be supported"
+        )
+    except ValueError:
+        raise ParsingError(
+            "Couldn't parse the principal point tag. Sensor might not be supported"
+        )
```
Comment (elliott-imhoff) [inline imgparse/imgparse.py:247]: These could be combined into one except block e.g. `except (KeyError, ValueError):`

**Review by elliott-imhoff — APPROVED** [review]
Looks good!

File: imgparse/_version.py (hunk 1 above)
Comment (njwitthoeft) [reply imgparse/_version.py]: Done https://github.com/SenteraLLC/py-image-metadata-parser/pull/70/commits/5f918bc6a026511259eca02bd3d884fa2aec8fef

File: imgparse/imgparse.py (hunk 2 above)
Comment (njwitthoeft) [reply imgparse/imgparse.py]: Done https://github.com/SenteraLLC/py-image-metadata-parser/pull/70/commits/5f918bc6a026511259eca02bd3d884fa2aec8fef

**Pull request merged**